src/components/EditProfile.vue

```diff
@@ -20,9 +20,10 @@ export default {
     const store = useStore()
     const user = computed(() => props.userData);
   
-    const {account} = useWeb3();
-  // username, description, twitter, website, sig
-    const isEditProfileButtonShown = computed(() => account && account.value && ((account.value.toLowerCase() === user.value?.wallet.toLowerCase()) || !user.value));
+    const { account } = useWeb3();
+    // username, description, twitter, website, sig
+    const isEditProfileButtonShown = computed(() => account.value && user.value?.wallet && account.value.toLowerCase() === user.value.wallet.toLowerCase() || !user.value);
```

Comment:
Please let me know what this change is? Is the idea just to include `user.value?.wallet`? If so, please leave the truthy check for `account` or change it to `account?.value`, also please leave the brackets ("brackets are free"), it makes it easier for someone for glancing at the code to see what the original grouping of logic was.

Reply:
undone this commit/change